Scripts/utils.py

```diff
@@ -152,18 +153,18 @@ def choice(message="", bypass=False):
   if bypass == True:
     return True
 
-  # While loop until valid input
-  valid = False
-  while valid == False:
-    response = input("\n" + message + f" ({F.LIGHTCYAN_EX}y{S.R}/{F.LIGHTRED_EX}n{S.R}): ").strip()
-    if response == "Y" or response == "y":
-      return True
-    elif response == "N" or response == "n":
-      return False
-    elif response == "X" or response == "x":
-      return None
-    else:
-      print("\nInvalid Input. Enter Y or N  --  Or enter X to return to main menu.")  
+  questions = [
```

Comment:
this should return True, False or None

Reply:
So the options should be `yes`, `no` and `quit/cancel`?

Reply:
yes